plugins/Toolbox/resources/qml/ToolboxCompatibilityChart.qml

```diff
@@ -36,8 +36,8 @@ Item
             Label
             {
                 anchors.verticalCenter: parent.verticalCenter
-                elide: styleData.elideMode
-                text: styleData.value
+                elide: styleData.elideMode || Text.ElideNone
```

Comment:
styleData is something built into QT's table views. is there a reason to add the `|| Text.ElideNone`? Seems like if styleData is also not sufficient for some reason (although I'm pretty sure it will *always* be set) that we should just be able to use `Text.ElideNone` in all cases?

Reply:
> (although I'm pretty sure it will always be set)

Apparently it isn't. Queue ubiquitous "Qt is garbage" comment...
I think styleData can become undefined if a model "up the stream" becomes undefined.

Reply:
Haha you know me so well.

But in that case let's just always set it to use Text.ElideRight. The data that will be here is quite predictable.

Reply:
Fixed in 4e30507